azure/scope/managedcontrolplane.go

```diff
@@ -558,6 +558,7 @@ func (s *ManagedControlPlaneScope) AgentPoolSpec() azure.AgentPoolSpec {
 		),
 		Mode:              s.InfraMachinePool.Spec.Mode,
 		AvailabilityZones: s.InfraMachinePool.Spec.AvailabilityZones,
+		NodeLabels:        s.InfraMachinePool.Spec.NodeLabels,
```

Comment:
does this need to be added above in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1927/files#diff-cfd94adbc2ff767742e457e64836e93e88c6e03c3a64c04619c54b7ed76f284eR503 ?

Reply:
yes! I will do that for rest of PRs also